Exception 'org.postgresql.util.PSQLException: FEHLER: Relation »ext_od_output« existiert bereits' occured while building the ext-od-output.
Quitting on error...
